content/en/agent/guide/private-link.md

```diff
@@ -14,6 +14,10 @@ further_reading:
 Datadog exposes AWS PrivateLink endpoints in <b>us-east-1</b>.
 </div>
 
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">The Datadog PrivateLink integration does not support the Datadog for Government site.</div>
```

Comment:
nit: I don't think 'integration' is the right word. Probably best to just say 'The Datadog for Government site does not support Private Link' or something along those lines.